package.json

```diff
@@ -69,6 +70,7 @@
     ]
   },
   "devDependencies": {
+    "@types/react-gauge-chart": "^0.2.0",
```

Comment:
Why is the devDependencies version different than the dependencies version? I think you should be able to delete this line

Reply:
The types weren't saved in normal dependencies at all. I accidentally saved them to dev instead. They're moved now.